use_cases/privacy_protection_people_screens.yml

```diff
@@ -0,0 +1,18 @@
+nodes:
+- input.visual:
+    source: <path/to/video>
+- model.mask_rcnn:
+    detect: ["tv", "laptop"]
+- draw.instance_mask:
+    effect: {blur: 50}
+- model.yolact_edge:
+    model_type: r101-fpn
+    detect: ["person"]
+- dabble.bbox_count
+- draw.instance_mask:
+    effect: {blur: 50}
+- draw.bbox:
+    show_labels: True
+- draw.legend:
+    show: ["count"]
+- output.screen
```

Comment:
Add a `.yml` for system testing of use cases as well. See other yamls in this [folder](https://github.com/aimakerspace/PeekingDuck/tree/main/tests/use_cases) for example. The input source can be an image from the `tests` folder, and `output.screen` should be excluded.

Reply:
Added the `.yml` file for system testing